packages/@n8n/nodes-langchain/nodes/tools/ToolWorkflow/v2/utils/WorkflowToolService.ts

```diff
@@ -54,23 +53,25 @@ export class WorkflowToolService {
 		name,
 		description,
 		itemIndex,
+		returnAllItems,
 	}: {
 		name: string;
 		description: string;
 		itemIndex: number;
+		returnAllItems?: boolean;
```

Comment:
Nitpick but IMO it would be nicer to have this as an `options` property in constructor since only `executeSubWorkflow` needs the value. Now we're just passing it through in `createTool` and `runFunction`.